warning: left shift count >= width of type [-Wshift-count-overflow]

A warning  occurs while compiling the example sketch on x86 Windows 10 and Arduino IDE 2.2.1

```text
In file included from d:\Arduino_APA102\src\Arduino_APA102.cpp:20:0:
d:\Arduino_APA102\src\Arduino_APA102.h: In member function 'uint32_t Arduino_APA102::Color(uint8_t, uint8_t, uint8_t)':
d:\Arduino_APA102\src\Arduino_APA102.h:49:33: warning: left shift count >= width of type [-Wshift-count-overflow]
     return (uint32_t)(newRed << 16 | newGreen << 8 | newBlue);
```

--- a/src/Arduino_APA102.h
+++ b/src/Arduino_APA102.h
@@ -46,7 +46,7 @@ public:
   
   
   uint32_t Color(uint8_t newRed, uint8_t newGreen, uint8_t newBlue) {
-    return (uint32_t)(newRed << 16 | newGreen << 8 | newBlue);
+    return (uint32_t)(((uint32_t)newRed) << 16 | ((uint32_t)newGreen) << 8 | newBlue);
   }
 
   uint16_t getLeds(){

